dallinger/command_line.py

```diff
@@ -318,6 +319,20 @@ def deploy_sandbox_shared_setup(verbose=True, app=None, web_procs=1, exp_config=
     (id, tmp) = setup_experiment(debug=False, verbose=verbose, app=app,
                                  exp_config=exp_config)
 
+    # Register the experiment on OSF
+    # Load configuration.
+    config = get_config()
+    if not config.ready:
+        config.load_config()
+
+    try:
+        config.get('osf_access_token')
+        log("Preregistering experiment on OSF...")
+        registration.register(id, snapshot=None)
+
+    except Exception:
+        pass
```

Comment:
This isn't quite as bad as a bare `except:` in Python, but it's still something to avoid. If you're anticipating some particular problem, be specific about it. If you're not, don't try to catch everything, and then we'll get tracebacks that actually help us find the problem if something goes wrong. If you don't want to show the user a traceback, at least log a warning that something didn't work so we don't just swallow unanticipated exceptions silently.